choc-pizza.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			Chocolate Pizza
+		</title>
+		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
+		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
+		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand' rel='stylesheet' type='text/css' />
+	</head>
+	<body class="container_16 clearfix">
+		<header id="top-header" class="grid_15">
+			<div id="top-logo" class="toplogo">
+				<img src="img/logo.png" />
```

Comment:
I had a reason at the time for aligning everything along grid_15, but now I
can't remember what it was. I will change those to grid_16 and see what
happens and make adjustments as necessary. Thank you.

On Mon, Jan 19, 2015 at 2:33 PM, Dexter Lesaca <notifications@github.com>
wrote:

> In choc-pizza.html
> <https://github.com/jayspringate/chocolate-pizza/pull/2#discussion-diff-23191516>
> :
>
> > @@ -0,0 +1,119 @@
> > +<!DOCTYPE html>
> > +<html>
> > +	<head>
> > +		<title>
> > +			Chocolate Pizza
> > +		</title>
> > +		<link href="css/reset-choc.css" type="text/css" rel="stylesheet" />
> > +		<link href="css/style-choc.css" type="text/css" rel="stylesheet" />
> > +		<link href="css/960_16_col.css" type="text/css" rel="stylesheet" />
> > +		<link href='http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>' rel='stylesheet' type='text/css' />
> > +	</head>
> > +	<body class="container_16 clearfix">
> > +		<header id="top-header" class="grid_15">
> > +			<div id="top-logo" class="toplogo">
> > +				<img src="img/logo.png" />
>
> Both are acceptable. If you’re going to apply the image, use the
> background image property in css. Just be sure to apply it to an existing
> element, not create a new one. — Sent from Mailbox
> … <#14b0454984a19435_>
> On Mon, Jan 19, 2015 at 2:32 PM, Jay Springate <notifications@github.com>
> wrote: >> @@ -0,0 +1,119 @@ >> +<!DOCTYPE html> >> +<html> >> + <head> >> +
> <title> >> + Chocolate Pizza >> + </title> >> + <link
> href="css/reset-choc.css" type="text/css" rel="stylesheet" /> >> + <link
> href="css/style-choc.css" type="text/css" rel="stylesheet" /> >> + <link
> href="css/960_16_col.css" type="text/css" rel="stylesheet" /> >> + <link
> href='
> http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand
> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>'
> rel='stylesheet' type='text/css' /> >> + </head> >> + <body
> class="container_16 clearfix"> >> + <header id="top-header"
> class="grid_15"> >> + <div id="top-logo" class="toplogo"> >> + <img
> src="img/logo.png" /> > Also, I never figured out the ideal way to
> implement that slash border. I > ended up screenshotting it and including
> it as an image in the html. I > realize this is incorrect because it
> includes styling in the html. Would > you prefer I approximate that border
> using one of the existing CSS styles, > or that I include the image purely
> through CSS somehow? Or is there another > solution? > On Mon, Jan 19, 2015
> at 2:25 PM, Dexter Lesaca <notifications@github.com> > wrote: > In
> choc-pizza.html > <
> https://github.com/jayspringate/chocolate-pizza/pull/2#discussion-diff-23191073>
> > : > > > @@ -0,0 +1,119 @@ > > +<!DOCTYPE html> > > +<html> > > + <head> >
> > + <title> > > + Chocolate Pizza > > + </title> > > + <link
> href="css/reset-choc.css" type="text/css" rel="stylesheet" /> > > + <link
> href="css/style-choc.css" type="text/css" rel="stylesheet" /> > > + <link
> href="css/960_16_col.css" type="text/css" rel="stylesheet" /> > > + <link
> href='
> http://fonts.googleapis.com/css?family=Yellowtail|Abel|Droid+Serif:400,700|Patrick+Hand
> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>
> <
> http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand>
> <http://fonts.googleapis.com/css?family=Yellowtail%7CAbel%7CDroid+Serif:400,700%7CPatrick+Hand%3E>'
> rel='stylesheet' type='text/css' /> > > + </head> > > + <body
> class="container_16 clearfix"> > > + <header id="top-header"
> class="grid_15"> > > + <div id="top-logo" class="toplogo"> > > + <img
> src="img/logo.png" /> > > "/" is not necessary for self-closing tags. > > —
> > Reply to this email directly or view it on GitHub > <
> https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191073>.
> > --- Reply to this email directly or view it on GitHub:
> https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191424
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jayspringate/chocolate-pizza/pull/2/files#r23191516>.
>